src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double topSide;
+    private double bottomSide;
+    private double lateralSide;
+
+    public IsoscelesTrapezoid(String color, double topSide, double bottomSide, double lateralSide) {
+        super(color);
+        this.topSide = topSide;
+        this.bottomSide = bottomSide;
+        this.lateralSide = lateralSide;
+    }
+
+    @Override
+    public double getArea() {
+        double a = this.topSide;
```

Comment:
what are a, b, c ?

Reply:
> what are a, b, c ?

<img width="677" alt="image" src="https://github.com/user-attachments/assets/d4e61a18-b292-435b-bcbd-7d1846c34aa6">

I suppose, as I added only Override annotation in the last commit, GitHub didn't display the rest of the code

Reply:
But if you mean, what are they in terms of code-readability - simplified namings for the formula. If it's a bad idea, I can rename or even delete them.